README.md

```diff
@@ -1,140 +1,126 @@
-- Checking for any large files (over 5MB) being committed; and
-- Cleaning Jupyter notebooks, which means removing all outputs and execution counts.
+To get started your system should meet the following requirements:
 
-We have configured `pre-commit` to run automatically on _every commit_. By running on each commit, we ensure
-that `pre-commit` will be able to detect all contraventions and keep our repo in a healthy state.
+1. Unix-based system (macOS, Linux, ...)
+2. Python 3.5+ installed
+3. The [`cookiecutter` package installed](#installing-cookiecutter)
 
-In order for `pre-commit` to run, action is needed to configure it on your system.
-- [Install](#installing-python-packages) the `pre-commit` package into your Python environment from
-`requirements-dev.txt`; and
-- Run `pre-commit install` to set-up `pre-commit` to run when code is _committed_.
+#### Installing cookiecutter
 
-### Setting up a baseline for the `detect-secrets` hook (if one doesn't already exist)
+There are many ways to install the [`cookiecutter`][cookiecutter] package. **Our recommendation is to install it at the
+system or user level**, rather than as a Python package via `pip` or `conda`. This ensures it is isolated from the rest
+of your system, and always available.
 
-The `detect-secrets` hook requires that you generate a baseline file if one is not already present within the root
-directory. This is done via running the following at the root of the repo:
+For macOS, install `cookiecutter` via [Homebrew][homebrew]:
 
-```shell script
-detect-secrets scan > .secrets.baseline
+```shell
+brew install cookiecutter
 ```
 
-Next, audit the baseline that has been generated by running:
+For Debian/Ubuntu, use the following commands:
 
-```shell script
-detect-secrets audit .secrets.baseline
+```shell
+sudo apt-get install cookiecutter
 ```
 
-When you run this command, you'll enter an interactive console and be presented with a list of high-entropy string /
-anything which _could_ be a secret, and asked to verify whether or not this is the case. By doing this, the hook will
-be in a position to know if you're later committing any _new_ secrets to the repo and it will be able to alert you
-accordingly.
+Otherwise, you can install `cookiecutter` via `pip` — you may wish to create a virtual environment first:
+
+```shell
+python3 -m pip install --user cookiecutter
+```
 
-### If `pre-commit` detects secrets during commit:
+## Changes to make post-creation
 
-If `pre-commit` detects any secrets when you try to create a commit, it will detail what it found and where to go to
-check the secret.
+Here's a few changes you **must** do once you've created your new project:
 
-If the detected secret is a false-positive, you should update the secrets baseline through the following steps:
+- Set up a Python virtual environment — there are **many** ways to set up a virtual environment, so we'll let you
+  decide what's best for you!
```

Comment:
Should we give examples of some ways in case people who are a bit new to virtual environments want some guidance? Aware we're not really targeting them but we could maintain neutrality and be helpful by saying something like:

> *For example, we use [venv](https://docs.python.org/3/tutorial/venv.html).

Reply:
We're going to explore which Python virtual environment that we'll recommend in `govcookiecutter`, so I think we'll leave this for now until we've done a bit more research! I'll add a link however to this [Pluralsight Tech Blog](https://www.pluralsight.com/tech-blog/managing-python-environments/) that @exfalsoquodlibet mentioned in the past.

Changes made in commit [`c715927`](https://github.com/ukgovdatascience/govcookiecutter/pull/10/commits/c71592757be0e1dd4dae0ce7c4bdfd63b56f1134). Marking as **Resolved**.